core/admin/mailu/configuration.py

```diff
@@ -18,7 +18,8 @@
     'DB_PW': None,
     'DB_HOST': 'database',
     'DB_NAME': 'mailu',
-    'SQLALCHEMY_DATABASE_URI': 'sqlite:////data/main.db',
+    'SQLITE_DATABASE_FILE':'data/main.db'
+    'SQLALCHEMY_DATABASE_URI': '',
```

Comment:
Tests fail on this line:
https://travis-ci.org/Mailu/Mailu/builds/480729143#L3158

Reply:
OK. I updated the patch to leave the SQLite URI as the value for SQLALCHEMY_DATABASE_URI